src/JsonSchema/SchemaStorage.php

```diff
@@ -27,29 +31,13 @@ public function getUriRetriever()
 	}
 
 	/**
-	 * @param UriRetrieverInterface $uriRetriever
-	 */
-	public function setUriRetriever($uriRetriever)
```

Comment:
@bighappyface you right, it's not only about circular references. I also wanted to simplify the interface. Now there is no RefResolvers you need to know about, until you have to